expyfun/tests/test_eyelink_controller.py

```diff
@@ -46,6 +47,9 @@ def test_eyelink_methods():
         # run much of the calibration code, but don't *actually* do it
         el._fake_calibration = True
         el.calibrate(beep=False, prompt=False)
+        fake_button_press(ec, 'c')
+        el.check_recalibrate(prompt=False)
```

Comment:
Shouldn't you assert that this is True ?

Reply:
The test gets hung up on the prompt screen. Could put in a ``fake_button_press`` if you want to test the prompt?

Reply:
I don't understand. It currently gets stuck, or will get stuck with my suggested addition of an `assert ` in this line preceding the `el...`?

Reply:
Read too quickly, you are correct